packages/blade/src/components/Input/OTPInput/__tests__/__snapshots__/OTPInput.native.test.tsx.snap

```diff
@@ -213,29 +213,24 @@ exports[`<OTPInput /> should render 1`] = `
             >
               <View
                 currentInteraction="default"
-                data-blade-component="base-box"
```

Comment:
@saurabhdaware not using blade box here anymore?

Reply:
No. The wrapper needs to be animated in certain cases so it uses Animated.View. I have removed animation for other inputs so shouldn't change anything visually.